doc/api/http.md

```diff
@@ -744,6 +744,35 @@ added: v0.4.0
 
 * {string} The request path.
 
+### `request.method`
+<!-- YAML
+added: v14.4.0
```

Comment:
`request.method` has been around for much, much longer than v14.4.0. You will probably have to do some digging with `git blame` to find out when the property was introduced, which can be easily done from within the Github web UI.

Reply:
I digging it code, finally find it. thank you for your patience guidance.
https://github.com/nodejs/node/commit/15ec99ec5945c07616cd9299e126e9919bc9c8cc#diff-1c0f1c434b17b7f8795d44a51a14320aR507